.gitignore

```diff
@@ -17,4 +17,4 @@ edgesite/edgesite
 keadm/keadm
 _output
 _tmp
-build/edge/tmp
+build/edge/tmp
```

Comment:
looks like unnecessary change here? or git does a funny thing?

Reply:
I think it's auto-generated and I didn't make any changes to it. Maybe it's because I introduced unnecessary configurations when I clone the project with the cloud code platform before and re-clone it locally. But it was not an intentional event.